web/impact/impact/tests/test_organization_history_view.py

```diff
@@ -333,6 +334,45 @@ def test_startup_became_finalist_no_created_at(self):
             self.assertEqual(1, len(events))
             self.assertEqual(start_date, events[0]["datetime"])
 
+    def test_startup_became_winner(self):
+        startup = StartupFactory()
+        startup_status = StartupStatusFactory(
+            program_startup_status__startup_role__name=StartupRole.SILVER_WINNER,
+            startup=startup)
+        program = startup_status.program_startup_status.program
+        role = startup_status.program_startup_status.startup_role
+        with self.login(email=self.basic_user().email):
+            url = reverse(OrganizationHistoryView.view_name,
+                          args=[startup.organization.id])
+            response = self.client.get(url)
+            events = find_events(response.data["results"],
+                                 OrganizationBecameWinnerEvent.EVENT_TYPE)
+            self.assertEqual(1, len(events))
+            self.assertTrue(program.name in events[0]["description"])
+            self.assertEqual(program.name, events[0]["program"])
+            self.assertEqual(program.id, events[0]["program_id"])
+            self.assertEqual(program.cycle.name, events[0]["cycle"])
+            self.assertEqual(program.cycle.id, events[0]["cycle_id"])
+            self.assertEqual(role.name, events[0]["winner_level"])
+
+    def test_startup_became_winner_no_created_at(self):
+        startup = StartupFactory()
+        end_date = days_from_now(-10)
+        startup_status = StartupStatusFactory(
+            program_startup_status__startup_role__name=StartupRole.GOLD_WINNER,
+            program_startup_status__program__end_date=end_date,
+            startup=startup)
+        startup_status.created_at = None
+        startup_status.save()
+        with self.login(email=self.basic_user().email):
```

Comment:
likewise, why a `with`?

Reply:
I was trying to keep the test consistent with the ones written previously.

Reply:
Oh, okay, I see. It's that TestPlus nonsense. Okay, not your fault.